man/calcEmiTarget.Rd

```diff
@@ -21,15 +21,9 @@ turned off for inpudata generation}
 \description{
 This function calculates the emissions targets for the NDC scenarios applied in the REMIND module 45_carbonprice realization NDC.
 It contains the following steps:
-\enumerate{
-\item Read country-level NDC targets as target factors (target year emissions normalized by 2015 emissions).
-\item Extrapolate NDC targets from 2030 to 2035 for countries which do not have 2035 NDC targets (yet).
-\item Aggregate country-level target factors to region-level target factors using 2015 emissions as weight ("Ghgfactor").
-\item Calculate share of emissions covered under NDC target per REMIND region ("Ghgshare").
 The parameters calculated in 3.) and 4.) are further used in the NDC realization to calculate the region-wide NDC emissions targets
 in terms of total GHG emissions excl. bunkers and excl. LULUCF sectors.
 }
-}
```

Comment:
That's just some cleaning and not related to the AP restructuring, right? Otherwise I don't get it :)